README.md

```diff
@@ -67,7 +67,7 @@ A collection of awesome browser-side [JavaScript](https://developer.mozilla.org/
   * [SDK](#sdk)
   * [Misc](#misc)
   * [Podcasts](#podcasts)
-* [Worth Reading](#worth-reading)
+* [Reading](#worth-reading)
```

Comment:
I would like to accept this, however, can you please take this change out as it's out of scope for this PR.
If you want to propose on renaming `Worth Reading` to `Reading`, please create a dedicated PR and make sure to rename the link as well. But I honestly prefer it as Worth Reading.